packages/app/src/app/pages/Client/Prompt/index.tsx

```diff
@@ -0,0 +1,77 @@
+import { Button } from '@codesandbox/components';
+import React, { FunctionComponent, useEffect, useState } from 'react';
+
+import { SubTitle } from 'app/components/SubTitle';
+import { Title } from 'app/components/Title';
+import { useAppState } from 'app/overmind';
+
+import { SignInModalElement } from 'app/pages/SignIn/Modal';
+import { Buttons, Container } from './elements';
+
+export const Prompt: FunctionComponent = () => {
+  const { authToken, error, isLoadingCLI, user } = useAppState();
+
+  const [deepLink, setDeepLink] = useState('');
+
+  useEffect(() => {
+    setDeepLink(`codesandbox://auth-completion/?nonce=${authToken}`);
+  }, [authToken]);
+
+  if (error) {
+    return (
+      <Container>
+        <Title>An error occured:</Title>
+
+        <SubTitle>{error}</SubTitle>
+
+        <Buttons>
+          <Button autoWidth href="/?from-app=1">
+            Go to homepage
+          </Button>
+        </Buttons>
+      </Container>
+    );
+  }
+
+  if (!user?.username) {
+    return (
+      <Container>
+        <Title>Welcome to CodeSandbox!</Title>
+
+        <SubTitle style={{ paddingBottom: 16 }}>You need to sign in.</SubTitle>
+
+        <SignInModalElement />
+      </Container>
+    );
+  }
+
+  if (isLoadingCLI) {
+    return (
+      <Container>
+        <Title>Fetching authorization key...</Title>
+      </Container>
+    );
+  }
+
+  return (
+    <Container>
+      <Title>Hello {user.username}!</Title>
+
+      <SubTitle>
+        <br />
+        Click the button below to continue and finish signing in.
+      </SubTitle>
+
+      <Buttons>
+        <Button
+          as="a"
+          autoWidth
+          href={deepLink}
```

Comment:
If you want you can probably also directly put the link in here:
```suggestion
          href={`codesandbox://auth-completion/?nonce=${authToken}`}
```

Reply:
I think we can align to the current brand changing the typography and adding the logo. What do you think? 

![image](https://user-images.githubusercontent.com/1891339/119179977-02a86700-ba46-11eb-97da-cf6b1e83b402.png)